app/src/test/java/com/wiseassblog/kotlincalculator/CalculatorPresenterTest.kt

```diff
@@ -59,6 +59,10 @@ class CalculatorPresenterTest {
         //These are the assertions which must be satisfied in order to pass the test
         coVerify { eval.evaluateExpression(EXPRESSION) }
         coVerify { viewModel.setDisplayState(ANSWER) }
+
+        // but as well some not so important calls
+        excludeRecords { viewModel.setObserver(any()) }
+        confirmVerified(eval, viewModel)
```

Comment:
Makes final check that all recorded calls are covered with verification. Similar to `verifyNoMoreInteractions` in Mockito.